2024-09-08 15:57:27,449 - __main__ - ERROR - An error occurred: 'tuple' object is not callable
2024-09-08 16:02:59,323 - __main__ - ERROR - An error occurred: 'tuple' object is not callable
